### DISCUSSION TITLE
Web node complains error template: pattern matches no files: templates\approval.html

### DISCUSSION THREAD

**Discussion by kirankbs** [concourse/concourse#7708] [Help & Support]
Hello Folks, I am trying to set up a web node manually(no docker due to the old server) in the windows server. I am facing an error.

error: server: failed to load web static: parse files: template: pattern matches no files: templates\approval.html

I am using the latest Concurso distribution.

**Comment by taylorsilva** [comment]
Running the web node on windows is not something we test (windows machines are $$$) and we're mostly unix devs here. I don't think many (any?) users in the community do this either.

That said, let's see if we can help get a web node running on windows for you!
Which version of Concourse were you trying to run?
Where did you download the concourse binary from? The github release page? https://github.com/concourse/concourse/releases

Based on the error it seems like it's complaining about not finding the web files that **should** be packed inside the binary. Did you set `CONCOURSE_WEB_PUBLIC_DIR`/`--web-public-dir` flag? If you did then try removing it.

If you could share all the config options you passed to Concourse that may help us figure out what's wrong.
